potato/server_utils/cli_utlis.py

```diff
@@ -0,0 +1,37 @@
+import requests
+import zipfile
+import io
+
+# all the available examples in the project hub
+project_hub = {
+    'simple_schema_examples': 'https://github.com/davidjurgens/potato/raw/pypi/project-hub/simple_examples.zip',
+    'dialogue_analysis': 'https://github.com/davidjurgens/potato/raw/pypi/project-hub/dialogue_analysis.zip',
+    'empathy': 'https://github.com/davidjurgens/potato/raw/pypi/project-hub/empathy.zip',
+    'gif_reply': 'https://github.com/davidjurgens/potato/raw/pypi/project-hub/gif_reply.zip',
+    'immigration_framing': 'https://github.com/davidjurgens/potato/raw/pypi/project-hub/immigration_framing.zip',
+    'match_finding': 'https://github.com/davidjurgens/potato/raw/pypi/project-hub/match_finding.zip',
+    'match_finding_with_prestudy': 'https://github.com/davidjurgens/potato/raw/pypi/project-hub/match_finding_with_prestudy.zip',
+    'sentiment_analysis': 'https://github.com/davidjurgens/potato/raw/pypi/project-hub/sentiment_analysis.zip',
+    'summarization_evaluation': 'https://github.com/davidjurgens/potato/raw/pypi/project-hub/summarization_evaluation.zip',
+    'textual_uncertainty': 'https://github.com/davidjurgens/potato/raw/pypi/project-hub/textual_uncertainty.zip',
+    'question_answering': 'https://github.com/davidjurgens/potato/raw/pypi/project-hub/question_answering.zip'
+}
+
+# get a speicific project from the hub
+def get_project_from_hub(name):
```

Comment:
We probably want some checking here in case the user has already gotten the files to ask if they want to overwrite the contents. I could see someone editing the files and then accidentally running the command again and wiping out their changes. Also, do we want to give an option of where they want to save the results (with a default so they can just press 'y')

Reply:
fixed with [576e84d](https://github.com/davidjurgens/potato/pull/32/commits/576e84db7ee793cb9f73dfc692ecfabb609e6fcb)